sdks/js/src/types.ts

```diff
@@ -1757,15 +1757,15 @@ const DatasetSchema = z.object({
 });
 
 const AppTypeSchema = z.object({
-  id: ModelIdSchema,
+  id: ModelIdSchema.optional(),
```

Comment:
Any implication elsewhere to expect from putting those optional?

Reply:
The type is used for importing and exporting - on import it's optional on purpose, on export it will always be filled - can it be an issue for api users ? I can also duplicate the type ?